app/views/handlers/block.py

```diff
@@ -118,7 +120,20 @@ def _set_started_at_metadata(self):
             logger.info("Survey started", started_at=started_at)
             collection_metadata["started_at"] = started_at
 
-    def _get_safe_page_title(self, title):
+    def _get_safe_page_title(self, page_title):
         return safe_content(
-            f'{self._schema.get_single_string_value(title)} - {self._schema.json["title"]}'
+            f'{self._schema.get_single_string_value(page_title)} - {self._schema.json["title"]}'
         )
+
+    def _get_custom_page_title(self, page_title: str) -> str:
```

Comment:
It would be clearer if this was called something like `_resolve_custom_page_title`.

Reply:
Done